db/migrate/20221018144622_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :AuthorId, null: false, foreign_key: { to_table: :users }
+      t.string :title
+      t.text :text
+      t.integer :CommentCounter
+      t.integer :LikesCounter
```

Comment:
- [ ] Also same here it is better to write the name of the column `snake_case` not `UpperCamelCase`.

```ruby
t.integer :comments_counter
t.integer :likes_counter
```

Reply:
@ShahierNashaat !
Thanks for your feedback.